nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestLookupRecord.java

```diff
@@ -464,7 +464,9 @@ public void testAddFieldsToExistingRecordRouteToSuccess() throws InitializationE
         runner.run();
 
         final MockFlowFile out = runner.getFlowFilesForRelationship(LookupRecord.REL_SUCCESS).get(0);
-        out.assertContentEquals("John Doe,48,soccer,basketball\nJane Doe,47\n");
+        assertTrue(out.getContent().equals("John Doe,48,soccer,basketball\nJane Doe,47\n") 
+            || out.getContent().equals("John Doe,48,basketball,soccer\nJane Doe,47\n"));
```

Comment:
The test class should continue to use the `assertContentEquals()` method on `MockFlowFile`, as opposed to checking different values.

Reply:
Do you mean I should modify the `assertContentEquals()` method so it ignores the order?